frescobaldi_app/vcs/gitjob.py

```diff
@@ -182,27 +182,24 @@ def _finished(self, exitcode, exitstatus):
     def stdout(self):
         """
         Returns the content of the stdout output, if any.
+        
+        Returns: 
+            None: Git() is running / Git() hasn't started / Git() has crashed
+             b'': Git() has finished running and returns a binary result.
+              []: Git() has finished running and returns a string-list result.
         """
-        # TODO: should we check isRunning() before or can we rely on the "is not None" check?
-        # A simpler approach would be to simply return self._stdout and have the caller interpret
-        # the type of result: either None (job not completed) or a (potentially empty) string list
```

Comment:
I think it's better to return `_stdout` and `_stderr` directly to let the caller decide what to next.

So I pushed this commit.